ci/django.yml

```diff
@@ -0,0 +1,29 @@
+name: Django CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Can we use v2 of checkout? https://github.com/actions/checkout

Reply:
Done